tests/create_metric_filter/main.tf

```diff
@@ -18,8 +18,3 @@ locals {
   ]
 }
 
-module "create_metric_filter" {
```

Comment:
Let's restore this module. Removing the entire module invalidates the test. The test must invoke the module, or it is not actually testing anything.